subworkflows/local/profiling.nf

```diff
@@ -106,9 +100,8 @@ workflow PROFILING {
                                             // re-extract meta from file names, use filename without rma to
                                             // ensure we keep paired-end information in downstream filenames
                                             // when no pair-merging
-                                            def meta_new = meta.clone()
-                                            meta_new['db_name'] = meta.id
-                                            meta_new['id'] = rma.baseName
+                                            def meta_new = meta + [db_name: meta.id, id: rma.baseName]
+
                                         [ meta_new, rma ]
```

Comment:
or you can just do:
```
[ meta + [db_name: meta.id, id: rma.baseName], rma ]
```

Reply:
I personally prefer to not embed operations within the output declaration, so I'll leave this one off